src/clients/mattermost.ts

```diff
@@ -1,8 +1,10 @@
 import Client from 'mattermost-redux/client/client4.js';
 
-export const newClient = (url: string, token: string): Client => {
+import {configStore} from '../store';
+
+export const newMMClient = (token: string): Client => {
```

Comment:
Does mm clients allow passing in options object like zd client.  Then can omit seturl call

Reply:
yes. who cares?